dcos/package.py

```diff
@@ -169,3 +169,34 @@ def merge_installed(apps, subcommands, app_only, cli_only):
         merged.extend(indexed_subcommands.values())
 
     return merged
+
+
+def get_cosmos_url():
+    """
+    :returns: cosmos base url
+    :rtype: str
+    """
```

Comment:
Didn't we move this from `.../package/main.py` or something like that? Let's delete that code and change all of the call sites.